build.gradle

```diff
@@ -1,101 +1,94 @@
 buildscript {
-    ext.kotlin_version = '1.2.21'
-    ext.springBootVersion = '1.5.9.RELEASE'
-    ext.auroraStartersVersion = '3.5.0'
-    ext.repos = {
-        maven {
-            url "http://repo1.maven.org/maven2"
-        }
-
-        maven {
-            url "http://jcenter.bintray.com/"
-        }
-
-      mavenLocal()
+  ext.kotlin_version = '1.2.30'
+  ext.springBootVersion = '2.0.0.RELEASE'
+  ext.auroraStartersVersion = '1.0.0'
+  ext.repos = {
+    maven {
+      url "http://repo1.maven.org/maven2"
     }
 
-    repositories repos
-
-    dependencies {
-        classpath(
-                "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version",
-                "org.jetbrains.kotlin:kotlin-noarg:$kotlin_version",
-                "org.jetbrains.kotlin:kotlin-allopen:$kotlin_version",
-                "org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion",
-                'no.skatteetaten.aurora.gradle.plugins:aurora-gradle-plugin:1.2.1',
-        )
+    maven {
+      url "http://jcenter.bintray.com/"
     }
-}
 
-plugins {
-    id "com.gorylenko.gradle-git-properties" version "1.4.17"
+    mavenLocal()
+  }
+
+  repositories repos
+
+  dependencies {
+    classpath(
+        "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version",
+        "org.jetbrains.kotlin:kotlin-noarg:$kotlin_version",
+        "org.jetbrains.kotlin:kotlin-allopen:$kotlin_version",
+        "org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion",
+        'no.skatteetaten.aurora.gradle.plugins:aurora-gradle-plugin:1.2.1',
```

Comment:
Jeg stemmer for at vi kun bruker variable for versjoner der hvor vi bruker dem mer enn 1 plass.

Reply:
SpringBootVersion er også kun brukt 1 plass